.gitignore

```diff
@@ -14,6 +14,13 @@
 /coverage/*
 /libpeerconnection.log
 npm-debug.log*
+yarn-error.log
 testem.log
+.idea
```

Comment:
What is this `.idea` file?

Reply:
its the folder where IntelliJ its project-specific configs. It's nice to ignore it since we have a couple of external contributors here :smile: